example/src/screens/TrustmarkScreen.tsx

```diff
@@ -0,0 +1,200 @@
+          onPress: () =>
+            navigation.navigate("TrustmarkQrCode", { credentialType }),
+          isLoading: false,
+          hasError: { status: false, error: undefined },
+          isDone: false,
+          icon: iconByCredentialType[credentialType],
+        })),
+    [navigation, credentials]
+  );
+
+  return (
+    <View style={styles.container}>
+      <FlatList
+        style={styles.list}
+        contentContainerStyle={{
+          margin: IOVisualCostants.appMarginDefault,
+        }}
+        data={scenarios}
+        keyExtractor={(item, index) => `${item?.title}-${index}`}
+        renderItem={({ item }) => (
+          <>
+            {item && (
+              <>
+                <TestScenario
+                  onPress={item.onPress}
+                  title={item.title}
+                  isLoading={item.isLoading}
+                  hasError={item.hasError}
+                  isDone={item.isDone}
+                  icon={item.icon}
+                  isPresent={item.isPresent}
+                />
+                <VSpacer />
+              </>
+            )}
+          </>
+        )}
+      />
+    </View>
+  );
+};
+
+/**
+ * This screen is for displays a credential trustmark
+ */
+export const TrustmarkQrCodeScreen = ({
+  route,
+}: TrustmarkQrCodeScreenProps) => {
+  const { credentialType } = route.params;
+  const dispatch = useAppDispatch();
+  const trustmark = useAppSelector(selectTrustmark(credentialType));
+  const trustmarkAsyncStatus = useAppSelector(
+    selectTrustmarkAsyncStatus(credentialType)
+  );
+  const credential = useAppSelector(selectCredential(credentialType));
+  const selectedEnv = useAppSelector(selectEnv);
+  const { VERIFIER_BASE_URL } = getEnv(selectedEnv);
+
+  useDebugInfo({
+    trustmarkJwt: trustmark?.trustmarkJwt,
+    trustmarkAsyncStatus,
+  });
+
+  React.useEffect(() => {
+    if (credential) {
+      const documentNumber = getCredentialDocumentNumber(credential);
+      dispatch(getTrustmarkThunk({ credentialType, documentNumber }));
+
+      return () => {
+        dispatch(trustmarkReset(credentialType));
+      };
+    }
+
+    return undefined;
+  }, [dispatch, credential, credentialType]);
+
+  if (trustmark === undefined) {
+    return null;
+  }
+
+  const { trustmarkJwt } = trustmark;
+  const trustmarkUrl = `${VERIFIER_BASE_URL}?tm=${trustmarkJwt}`;
+
+  return (
+    <View style={styles.trustmarkContainer}>
+      <QrCodeImage value={trustmarkUrl} size={"90%"} />
+      <ExpirationTimer time={120} />
+    </View>
+  );
+};
+
+/**
+ * Returns the document number for a credential, if applicable
+ * @param credential the credential from which to extract the document number
+ * @returns a string representing the document number, undefined if not found
+ */
+const getCredentialDocumentNumber = (
+  credential: CredentialResult | undefined
+): string | undefined => {
+  if (credential === undefined) {
+    return undefined;
+  }
```

Comment:
Why not?
```suggestion
const getCredentialDocumentNumber = (
  credential: CredentialResult
): string | undefined => {
```

Reply:
Addressed here: https://github.com/pagopa/io-react-native-wallet/pull/156/commits/558242937799b2b389e788c18168467c19b20dc4